posthog/models/person_overrides/sql.py

```diff
@@ -141,3 +141,49 @@
     ON CLUSTER '{CLICKHOUSE_CLUSTER}'
     SYNC
 """
+
+GET_LATEST_PERSON_OVERRIDE_ID_SQL = f"""
+SELECT
+    team_id,
+    old_person_id,
+    override_person_id
+FROM
+    `{CLICKHOUSE_DATABASE}`.`person_overrides` AS overrides
+JOIN (
```

Comment:
Rather than joining here which could require creating a hash in memory if it doesn't get optimized away, can we instead use [`argMax`](https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/argmax/), or [`LIMIT abc BY 1`](https://clickhouse.com/docs/en/sql-reference/statements/select/limit-by/) ?

Reply:
Gave `argMax` a shot, I believe it works :+1: